docs/manual/dependency_generators.md

```diff
@@ -33,7 +33,7 @@ A file attribute is represented by a macro file in `%{_fileattrsdir}` (typically
 %__NAME_protocol
 ```
 
-NAME needs to be replaced by the name choosen for the file attribute and needs to be the same as the file name of the macro file itself (without the `.attr` suffix). While technically all of them are optional, typically two or more of them are present to form a meaningul attribute. All the values are further macro-expanded on use, and additionally, the path and magic related values are interpreted as extended regular expressions.
+NAME needs to be replaced by the name choosen for the file attribute and needs to be the same as the file name of the macro file itself (without the `.attr` suffix). While technically all of them are optional, typically at least two of them are present to form a meaninful attribute: `*_path` and/or `*_magic` to match any files at all and at least one generator. All the values are further macro-expanded on use, and additionally, the path and magic related values are interpreted as extended regular expressions.
```

Comment:
Seems like the "meaningful" word only got corrected into another typo :smile:

Reply:
*sigh*